bitnami/kafka/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kafka
-version: 1.10.1
+version: 1.11.0
```

Comment:
I think we should bump the major version

Reply:
Bumping major version